driver/src/models/mod.rs

```diff
@@ -1,6 +1,7 @@
 pub mod state;
 pub mod flux;
 pub mod order;
+#[macro_use]
```

Comment:
This appears out of nowhere? I also don't see any changes in this file that relate to the inclusion of this line. Was this intentional?

Reply:
How did you find this one? I had already pushed the cleaning commit a long time before your comment,